project proposal.ipynb

```diff
@@ -77,10 +77,10 @@
     "We almost lost hope - but eventually we managed to find two interesting sources of data. \n",
```

Comment:
<p>we can hide this, I remember Idan mentioned it in one of the lectures.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>